File: woodroid/src/main/java/me/gilo/woodroid/models/filters/CouponFilter.java
package me.gilo.woodroid.models.filters;

public class CouponFilter extends ListFilter{

    String code;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
        addFilter("code", code);
    }
}

File: woodroid/src/main/java/me/gilo/woodroid/models/filters/CouponFilter.kt
package me.gilo.woodroid.models.filters

class CouponFilter : ListFilter() {

    internal lateinit var code: String

    fun getCode(): String {
        return code
    }

    fun setCode(code: String) {
        this.code = code
        addFilter("code", code)
    }
}
